### PR TITLE
add phylop

### PR DESCRIPTION
**Pull request by qingl0331**
add readme

### CODE REVIEW

File: score-sets/GRCh37/sift/make.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+cd sift
+wget  ftp://ftp.jcvi.org/pub/data/sift/Human_db_37_ensembl_63/Human*sqlite.gz 
+gunzip *gz
+set -euo pipefail
+(echo -e "#chrom\tpos\tref\talt\tsiftScore";
+for f in `ls -v ./Human_CHR*.sqlite`; do
+    set -e
+    perl query.pl $f | sort --temporary-directory=. -k1,1 -k2,2n
+done ) | bgzip -c > sift.txt.gz
+tabix -b2 -e2 sift.txt.gz
```
Comment (brentp) [inline score-sets/GRCh37/sift/make.sh]: could you cleanup *.sqlite at the end?

**Comment by qingl0331** [comment]
sure~

2017-09-01 14:49 GMT-06:00 Brent Pedersen <notifications@github.com>:

**Comment by jimhavrilla** [comment]
for getting phyloP you can use http://hgdownload.cse.ucsc.edu/goldenpath/hg19/phyloP46way/primates/ to get the chromosome scores for phyloP in Wig format then use wigtoBigWig and bigWigtoBedGraph

**Comment by jimhavrilla** [comment]
additionally there is a newer, more up-to-date SIFT than the one on craig venter's website at http://sift.bii.a-star.edu.sg/sift4g/public/Homo_sapiens/GRCh37.74/
paper is here: http://sift.bii.a-star.edu.sg/www/nprot2016_vaser.pdf

**Comment by qingl0331** [comment]
Actually I have contacted the developer and they suggested us to use  the
latest mammalian scores

2017-09-01 15:20 GMT-06:00 Jim Havrilla <notifications@github.com>:

**Comment by jimhavrilla** [comment]
you mean the placentalmammal scores?

**Comment by qingl0331** [comment]
yep

2017-09-01 15:25 GMT-06:00 Jim Havrilla <notifications@github.com>:

**Comment by jimhavrilla** [comment]
then you can use the wigs in http://hgdownload.cse.ucsc.edu/goldenpath/hg19/phyloP46way/placentalMammals/ instead of primates and do the same thing unless there is a newer than 2009 one.  did you see the SIFT 4G link above?

**Comment by jimhavrilla** [comment]
Really?  Odd since they put out a new SIFT software last year...sqlite is
way faster no doubt though

Jim Havrilla
PhD Candidate in Human Genetics, University of Utah
Accelerated BS/MS in Biomedical Engineering, Drexel University '12,
Concentration: Bioinformatics
*"Memory, comprehension, communication, motivation"*

**Comment by jimhavrilla** [comment]
Did you push your make.sh for phyloP?

Jim Havrilla
PhD Candidate in Human Genetics, University of Utah
Accelerated BS/MS in Biomedical Engineering, Drexel University '12,
Concentration: Bioinformatics
*"Memory, comprehension, communication, motivation"*

**Comment by qingl0331** [comment]
Not yet, , trying to figure out how to do it in a separate branch

2017-09-01 15:37 GMT-06:00 Jim Havrilla <notifications@github.com>:

> Did you push your make.sh for phyloP?
>
> Jim Havrilla
> PhD Candidate in Human Genetics, University of Utah
> Accelerated BS/MS in Biomedical Engineering, Drexel University '12,
> Concentration: Bioinformatics
> *"Memory, comprehension, communication, motivation"*
>

**Comment by jimhavrilla** [comment]
It's pretty hard since you made all those commits in master.  Best thing to
do is rebase all the commits into one.  Then make your new branch.  Then
hard reset the HEAD to the last commit brent made in master, then checkout
to your new branch and it should have all those commits in one commit.
Then push that branch and create a pull request on it. I think that should
cover you.

Jim Havrilla
PhD Candidate in Human Genetics, University of Utah
Accelerated BS/MS in Biomedical Engineering, Drexel University '12,
Concentration: Bioinformatics
*"Memory, comprehension, communication, motivation"*

**Comment by jimhavrilla** [comment]
Refer to this:
https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git
And this:
https://stackoverflow.com/questions/1628563/move-the-most-recent-commits-to-a-new-branch-with-git

Jim Havrilla
PhD Candidate in Human Genetics, University of Utah
Accelerated BS/MS in Biomedical Engineering, Drexel University '12,
Concentration: Bioinformatics
*"Memory, comprehension, communication, motivation"*

**Comment by qingl0331** [comment]
cool! thnx! 👍

2017-09-01 15:43 GMT-06:00 Jim Havrilla <notifications@github.com>:

> Refer to this:
> https://stackoverflow.com/questions/5189560/squash-my-
> last-x-commits-together-using-git
> And this:
> https://stackoverflow.com/questions/1628563/move-the-
> most-recent-commits-to-a-new-branch-with-git
>
> Jim Havrilla
> PhD Candidate in Human Genetics, University of Utah
> Accelerated BS/MS in Biomedical Engineering, Drexel University '12,
> Concentration: Bioinformatics
> *"Memory, comprehension, communication, motivation"*
>

**Comment by jimhavrilla** [comment]
Sure, git is a pain

Jim Havrilla
PhD Candidate in Human Genetics, University of Utah
Accelerated BS/MS in Biomedical Engineering, Drexel University '12,
Concentration: Bioinformatics
*"Memory, comprehension, communication, motivation"*

**Pull request merged**